lib/create-environment-api.ts

```diff
@@ -770,6 +770,40 @@ export default function createEnvironmentApi(makeRequest: MakeRequest) {
       }).then((response) => wrapEntry(makeRequest, response))
     },
 
+    /**
+     * Get entry references
+     * @param entryId - Entry ID
+     * @param maxDepth - Level of the entry descendants from 1 up to 10 maximum
+     * @returns Promise of Entry references
+     * @example ```javascript
+     * const contentful = require('contentful-management');
+     *
+     * const client = contentful.createClient({
+     *  accessToken: '<contentful_management_api_key>
+     * })
+     *
+     * // Get entry references
+     * client.getSpace('<space_id>')
+     * .then((space) => space.getEnvironment('<environment_id>'))
```

Comment:
I think it'd be useful to have the `getEntry().references` example here too -- since we added to the `entities/entry` file